src/MoBi.Presentation/DTO/ObjectBaseDTO.cs

```diff
@@ -90,9 +99,15 @@ private static class AllRules
             .WithRule((dto, name) => dto.IsNameUnique(name))
             .WithError(AppConstants.Validation.NameAlreadyUsed);
 
+         private static IBusinessRule nameDoesNotContainIllegalCharacters { get; } = CreateRule.For<IObjectBaseDTO>()
```

Comment:
@Yuri05 Copied the rule from Core. I wanted to use the base object, but there are some stuff in this ObjectBaseDTO that I don't quite understand so I'd rather not mess stuff around too much